client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -31,10 +33,15 @@ export function normalizeRevision( revision ) {
 	}
 
 	return {
-		...revision,
+		...omit( revision, [ 'title', 'content', 'excerpt', 'date', 'date_gmt', 'modified', 'modified_gmt' ] ),
 		...flow(
 			pick( [ 'title', 'content', 'excerpt' ] ),
 			mapValues( ( val = {} ) => val.rendered )
+		)( revision ),
+		...flow(
+			pick( [ 'date_gmt', 'modified_gmt' ] ),
```

Comment:
I'm bewildered by how this is working, specifically the flow/currying applied to `pick`. In my own Node console it errors as I'd expect it to, because `pick` isn't supposed to return a function but rather the object subset:

```
n_ > _.flow( _.pick( 'foo' ) )( { foo: 'bar' } )
TypeError: Expected a function
```

Can you explain what I'm missing?